.github/buildomat/jobs/test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#:
+#: name = "test"
+#: variety = "basic"
+#: target = "lab-opte-0.22"
```

Comment:
We're just unloading the existing driver so switch to regular `lab` target? Or even `helios-latest` since it doesn't seem like we need to explicitly running on bare metal. That would free up the lab machines to run other CI.

Reply:
We can do that once there is a sufficiently up OS image that runs on non-metal.

Reply:
Is `helios-latest` not up to date enough? We can ask josh to push newer bits if needed